boto/gs/connection.py

```diff
@@ -47,6 +47,17 @@ def __init__(self, gs_access_key_id=None, gs_secret_access_key=None,
                  "google", Bucket,
                  suppress_consec_slashes=suppress_consec_slashes)
 
+    def _required_auth_capability(self):
+        """
+        Overrides the S3 version of this method to remove the
+        @detect_potential_s3sigv4 decorator to avoid sending s3 credentials to
```

Comment:
Nit: I think the credentials were still the `gs` creds, but they were being used to compute sigv4 signatures; doing this A) added a mixture of x-amz- and x-goog- headers into the request, and B) isn't supported by GCS (only the old style -- version 2? -- is supported currently).